### PR TITLE
Initial Discord bot implementation

### PR DESCRIPTION
**Pull request by akirabaruah**
Still very rough around the edges, but seems to work. Currently supports
sending a command starting with `!puzzles`. By default, the command
lists unsolved puzzles, but you can supply a subcommand like `!puzzles
stuck` to view all puzzles with STUCK status. Current commands:
    - `!puzzles`
    - `!puzzles unsolved`
    - `!puzzles solved`
    - `!puzzles stuck`

Still need to implement:
- [x] Links to puzzles in response
- [x] Status for puzzles (since "stuck" includes STUCK and EXTRACTION)
- [x] Answers for solved puzzles (maybe?)

For #214

### CODE REVIEW

**Comment by akirabaruah** [comment]
It works :) and runs on heroku :+1: see the `#discord-bot-spam` channel in our server.

**Comment by akirabaruah** [comment]
One limitation we'll need to resolve: Discord embeds (used by this bot) currently limit field values to 1024 characters. If there are lots of puzzles to list, we might need more characters. One simple solution might be to use regular text instead of embeds...but embeds look so nice :joy:

File: chat/management/commands/rundiscordbot.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            line += f" ([sheet]({p.sheet}))"
+        lines.append(line)
+    print(f"lines: {lines}")
+    embed.add_field(name=title, value="\n".join(lines))
+    await message.channel.send(embed=embed)
+
+
+async def send_help(message):
+    print("Sending help message")
+    embed = discord.Embed(title="Cardi-P", description="Ahoy mateys, puzzle bot here!")
```
Comment (rawxfish) [inline chat/management/commands/rundiscordbot.py:79]: Hahaha

**Review by rawxfish — APPROVED** [review]

**Review by rgossiaux — APPROVED** [review]
Yay!

We can figure out the BOT_ACTIVE_HUNT stuff after MH

**Pull request merged**